include/average_waiting_time.h

```diff
@@ -0,0 +1,15 @@
+/*@file average_waiting_time.h
+ *@brief Has the declaration for the average_waiting_time function
+ *@detail The function has one input parameter: time_difference which is the difference between
+ * the time passenger arrived and the time he got into the elevator
+ *@return float returns the average time taken by the passengers waiting to get into the elevator
+ */
+
+#ifndef AVERAGE_WAITING_TIME_H_INCLUDED
```

Comment:
usually the coding convention is to use __AVERAGE_WAITING_TIME_H__ as header gurad name. This is a small issue.